packages/smpc/setup.cfg

```diff
@@ -0,0 +1,159 @@
+# This file is used to configure your project.
+# Read more about the various options under:
```

Comment:
This feels a bit heavy. I think I see where you're going with this but might be good to try to avoid too much eng code for the next couple days